.github/actions/waffles/Dockerfile

```diff
@@ -1,5 +1,5 @@
 # Container image that runs your code
-FROM ubuntu:22.04@sha256:2b7412e6465c3c7fc5bb21d3e6f1917c167358449fecac8176c6e496e5c1f05f
+FROM python:3.12
```

Comment:
Hope everything will still run fine with waffles 🤞

Reply:
Curious to see as well if renovate will suggest us a proper sha to use in there, for best practices.